metasyn/__main__.py

```diff
@@ -19,6 +22,7 @@
 Usage: metasyn [subcommand] [options]
 
 Available subcommands:
+    create-meta - generate a GMF/json file.
```

Comment:
```suggestion
    create-meta - generate a GMF (.json) file.
```

A minor tweak to be more in line in how I commonly have worded it on the docs.